app/Exceptions/Handler.php

```diff
@@ -42,7 +42,7 @@ public function report(Exception $exception)
      * @param  \Exception  $exception
      * @return \Illuminate\Http\Response
      */
-    public function render($request, Exception $exception)
+    public function render($request, Exception $exception): \Illuminate\Http\Response
```

Comment:
Please include the type for this at the top.  I tend to prefer this so that when I look at all the `use` statements at the top of a class, I can have a pretty good idea of it's dependencies.